File: 0136-single-number/0136-single-number.py
class Solution(object):
    def singleNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        freq = {}

        for el in nums:
            freq[el] = freq.get(el, 0)+1

        for el in freq:

            if freq[el] == 1: return el

File: 0136-single-number/0136-single-number.cpp
class Solution {
public:
    int singleNumber(vector<int>& nums) {
        
        for(int i=0; i<nums.size(); i++){
            int count = 0;

            for(int j=0; j<nums.size(); j++){

                if(nums[i] == nums[j]){
                    count +=1;
                }
            }

            if(count == 1){
                return nums[i];
            }else{
                count = 0;
            }

        }

        return 0;
    }
};
